### DISCUSSION TITLE
Naming

### DISCUSSION THREAD

**Discussion by spaceape0815** [brianvoe/slim-select#556] [Ideas]
Guys - thanks for developing Slimselect! We are actually including it into our system, and I / we really like and appreciate it.

One thing though is (maybe because we´re german?) a little bit pretty much ... I dunno. But, given our history (that never happened and nobody knows anything about it of course), working with the ss all the time and even with "ss-values" ist really kinda weird. ;-)

Don´t feel critizised or anything, I just didn´t know where else to put this, but I had to get it out of my head. 

:-*

**Comment by brianvoe** [comment]
ss is an abbreviation for slim select, any other interpretation would be incorrect.
